tests/TestCase/ORM/Association/BelongsToManyTest.php

```diff
@@ -405,7 +405,7 @@ public function testFinderOption(): void
             'finder' => 'published',
         ]);
         $articles->updateAll(['published' => 'N'], ['id' => 1]);
-        $entity = $tags->get(1, ['contain' => 'Articles']);
+        $entity = $tags->get(1, ...['contain' => 'Articles']);
```

Comment:
Is this something we would want to do with rector? We could also use the rector rule I wrote for find() and apply it to get() as well.

Reply:
Yes, the same rule for `find()` can work for `get()` too. Will just have to account for `key` in options array is now `$cacheKey` as argument name.

Reply:
Renaming the key is something we should be able to do with rector. I can put that together this week.

Reply:
Thanks, this is ready to be merged then.